GetIntoTeachingApi/Utils/Env.cs

```diff
@@ -7,6 +7,7 @@ public class Env : IEnv
         public bool IsDevelopment => EnvironmentName == "Development";
         public bool IsProduction => EnvironmentName == "Production";
         public bool IsStaging => EnvironmentName == "Staging";
+        public bool IsTest => EnvironmentName == null;

```

Comment:
is that a string check for null?  should it be string.IsNullOrEmpty() perhaps?

Reply:
It'll never be an empty string in this case; there's no really nice way of checking if we're running in the test environment vs dev, but we explicitly set the env name for all other environments so the `null` check works here

Reply:
cool